### DISCUSSION TITLE
Only html displaying in generated report

### DISCUSSION THREAD

**Discussion by mnicolaides** [danielpalme/ReportGenerator#649] [General]
Tech stack: VS 2022, .NET 6, C#, xUnit 2.5.0, coverlet collector 6.0.0, Run Coverlet Report VS2022,  ReportGenerator 5.2.1.0.
We have been using ReportGenerator for almost a year without issue. A few weeks ago I noticed the report was not displaying the lines of code properly. What displays is just basic html and no C# code.

Has anyone seen this?

![image](https://github.com/danielpalme/ReportGenerator/assets/60892114/b2741a28-651f-40cc-bf23-2bff51f570b0)

**Comment by sungam3r** [comment]
This is sign in page. See #574
